departments.rb

```diff
@@ -0,0 +1,33 @@
+require './employees'
+
+class Department
+
+attr_reader :name, :roster
+
+  def initialize(name)
+    @name = name
+    @roster = []
+    @good_employee = []
+
+  end
+
+  def add_employee(employee)
+    @roster << employee
+  end
+
+  def total_salary
+    total = 0
+    @roster.each do |employee|
+      total += employee.salary
+    end
+  return total
+  end
+
+  def give_raises
```

Comment:
Yeah, it looks like you stopped here, since this method doesn't call give_raise on the good employees yet.